README.md

```diff
@@ -122,28 +122,20 @@ you can use this:
 # Pull in the records-mover library - be sure to run the pip install above first!
 from records_mover import Session
 from pandas import DataFrame
-import sqlalchemy
 
 session = Session()
+session.set_stream_logging()
```

Comment:
As-is, this sample code didn't log.  Let's give people a good starting point that they can debug from!